.github/workflows/PullRequest-IEFPolicies.yml

```diff
@@ -29,13 +29,13 @@ jobs:
           fetch-depth: 0
 
       - name: Install GitVersion
-        uses: gittools/actions/gitversion/setup@v1.1.1
```

Comment:
Experience from work says this is a dangerous update. Leave until code is stable.